packages/langium-railroad/src/grammar-railroad.ts

```diff
@@ -39,16 +39,30 @@ export interface GrammarDiagramOptions {
     javascript?: string;
 }
 
+function styling(options?: GrammarDiagramOptions) {
+    return `<style>
+    ${defaultCss}
+    </style>${options?.css ? `
+    <style>
+    ${options.css}
+    </style>` : ''}`;
+}
+
+/**
+ * Creates a whole HTML file that contains all railroad diagrams.
+ *
+ * @param grammar - GrammarAST.Grammar
+ * @param options? - GrammarDiagramOptions
+ * Use to add additional styling to all diagrams and additional behavior for the created HTML.
+ *
+ * @returns text - string
```

Comment:
Minor: We usually don't annotate our parameter or return type information, since we already generate `.d.ts` files for all JavaScript files.

Reply:
Removed parameter and return type annotations.